rts/System/LogOutput.cpp

```diff
@@ -84,38 +83,28 @@ static spring::unordered_map<std::string, int> GetEnabledSections() {
 	enabledSections = StringToLower(enabledSections);
 	enabledSections = StringStrip(enabledSections, " \t\n\r");
 
-	// make the last "section:level" substring findable
-	if (!enabledSections.empty() && enabledSections.back() != ',')
-		enabledSections += ",";
-
-	// n=1 because <enabledSections> always starts with a ',' (if non-empty)
-	for (size_t n = 1; n < enabledSections.size(); ) {
-		const size_t k = enabledSections.find(',', n);
-
-		if (k != std::string::npos) {
-			const std::string& sub = enabledSections.substr(n, k - n);
-
-			if (!sub.empty()) {
-				const size_t sepChr = sub.find(':');
-
-				const std::string& logSec = (sepChr != std::string::npos)? sub.substr(         0,            sepChr): sub;
-				const std::string& logLvl = (sepChr != std::string::npos)? sub.substr(sepChr + 1, std::string::npos):  "";
-
-				if (!logLvl.empty()) {
-					sectionLevelMap[logSec] = StringToInt(logLvl);
-				} else {
-					#if defined(DEBUG)
-					sectionLevelMap[logSec] = LOG_LEVEL_DEBUG;
-					#else
-					sectionLevelMap[logSec] = DEFAULT_LOG_LEVEL;
-					#endif
+	for (const auto& subView: enabledSections | std::views::split(',')) {
+		auto sub = std::string_view(subView.begin(), subView.end());
```

Comment:
Apparently this also works:
```suggestion
		auto sub = std::string_view(subView);
```